packages/@sfcc-core/core/src/index.ts

```diff
@@ -2,13 +2,15 @@ export {
     ApiConfig,
     Logger,
     Extension,
+    Config,
     Service,
     ExtensionFactory,
     LogLevel,
 } from './types';
 
 export {
     core,
+    Core,
```

Comment:
What is the difference between these two?

Reply:
One is an instance, the other is the class.